instrumentation/lettuce/lettuce-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_0/LettuceDbAttributesGetter.java

```diff
@@ -21,30 +22,33 @@ final class LettuceDbAttributesGetter implements DbClientAttributesGetter<RedisC
       RedisCommandSanitizer.create(AgentCommonConfig.get().isStatementSanitizationEnabled());
 
   @Override
-  public String getSystem(RedisCommand<?, ?, ?> request) {
-    return DbIncubatingAttributes.DbSystemValues.REDIS;
+  public String getDbSystem(RedisCommand<?, ?, ?> redisCommand) {
+    return REDIS;
   }
 
+  @Deprecated
   @Override
   @Nullable
   public String getUser(RedisCommand<?, ?, ?> request) {
     return null;
   }
 
-  @Override
   @Nullable
-  public String getName(RedisCommand<?, ?, ?> request) {
+  @Override
```

Comment:
can you do a look through remaining code and revert the annotation ordering changes? sorry, less diffs really helps when reviewing these big PRs

Reply:
spotless should have taken care of this.. i see mixed ordering are used everywhere. does the order matter?

Reply:
> does the order matter?

no, but you changed the existing order, which makes for more diffs which makes these really big PRs harder to review

Reply:
this has been updated. it was renaming the method, not sure why i would intentionally change the order of annotation in this case.

Reply:
I pushed a commit to fix up a few more